### DISCUSSION TITLE
Enlarge text at device level?

### DISCUSSION THREAD

**Discussion by Tangible23** [totaldebug/atomic-calendar-revive#1112] [Support] [answered]
This noob has successfully stumbled his way into getting Atomic Calendar to display the iCloud calendar on a wall-mounted iPad mini. It’s great! However, the text is too small for this 76 year-old. 

I tried using the iPad’s Accessibility settings to enlarge everything globally, but it apparently has no effect on the Atomic calendar card. Is there a way to make that happen? I’ve seen suggestions to “just use card mod”, but that appears to require a whole additional set of installs and configuring, plus a knowledge of CSS. Is there a simple way to accomplish this simple task?

**Comment by marksie1988** [comment, marked as answer]
Hi there, 

`card-mod` is the recommended way to customise the card, the reason for this is that there are potentially hundreds of customisations that people could want and I cant add all of these to the card as it would make it far too complex to maintain and for users to know the options they need. 

The code below will allow you to change the font-size using `card-mod`

```
card_mod:
  style: |
    ha-card {
      font-size: 20px
    }
```
